nexus/db-model/src/schema.rs

```diff
@@ -139,35 +139,28 @@ table! {
 table! {
     switch_port_settings_link_config (port_settings_id, link_name) {
         port_settings_id -> Uuid,
-        lldp_service_config_id -> Uuid,
         link_name -> Text,
         mtu -> Int4,
         fec -> crate::SwitchLinkFecEnum,
         speed -> crate::SwitchLinkSpeedEnum,
         autoneg -> Bool,
+        lldp_link_config_id -> Uuid,
     }
 }
 
 table! {
-    lldp_service_config (id) {
+    lldp_link_config (id) {
```

Comment:
This makes the per-link lldp config data directly visible from that link's configuration.  There was a level of indirection through a `service_config` which wasn't necessary.